src/pkg/packager/inspect.go

```diff
@@ -38,3 +42,53 @@ func (p *Packager) Inspect(includeSBOM bool, outputSBOM string) error {
 
 	return nil
 }
+
+// InspectOCIOutput is the output of the InspectOCI command
+type InspectOCIOutput struct {
+	Tags   []string `yaml:"tags"`
+	Latest struct {
+		Tag        string             `yaml:"tag"`
+		Descriptor ocispec.Descriptor `yaml:"descriptor"`
+	} `yaml:"latest"`
+}
+
+// InspectOCI inspects an OCI image and returns the tags and latest tag with descriptor
+func (p *Packager) InspectOCI() error {
+	message.Debug("packager.InspectOCI()")
+	ref, err := registry.ParseReference(strings.TrimPrefix(p.cfg.DeployOpts.PackagePath, "oci://"))
+	if err != nil {
+		return err
+	}
+
+	repo, ctx, err := utils.OrasRemote(ref)
```

Comment:
Should refactor `utils.OrasRemote` to return a single var: `remote` w/ `repo/dst` and `ctx` as properties.